### DISCUSSION TITLE
Could we remove the listed ones in the default star list?

### DISCUSSION THREAD

**Discussion by liuxd** [community/community#9152] [Other Feature Feedback, Questions, & Ideas]
Could we remove the listed ones in the default star list so that I know which ones are not in any customized list? or a filter to do so?

**Comment by Anthonyfrankudoka** [comment]
Yes that’s a good idea

**Comment by Anthonyfrankudoka** [comment]
txn = AssetConfigTxn(sender=accounts[1]['pk'],
                     sp=params,
                     total=10000,           // Fungible tokens have totalIssuance greater than 1
                     default_frozen=False,
                     unit_name="ALICE",
                     asset_name="Alice's Coins@arc3",
                     manager="",
                     reserve="",
                     freeze="",
                     clawback="",
                     url="https://path/to/my/fungible/asset/metadata.json",
                     metadata_hash=json_metadata_hash,
                     decimals=2)            // Fungible tokens typically have decimals greater than

**Reply by Anthonyfrankudoka** [reply]
> txn = AssetConfigTxn(sender=accounts[1]['pk'],
> sp=params,
> total=10000, // Fungible tokens have totalIssuance greater than 1
> default_frozen=False,
> unit_name="ALICE",
> asset_name="Alice's Coins@arc3",
> manager="",
> reserve="",
> freeze="",
> clawback="",
> url="https://path/to/my/fungible/asset/metadata.json",
> metadata_hash=json_metadata_hash,
> decimals=2) // Fungible tokens typically have decimals greater than



> txn = AssetConfigTxn(sender=accounts[1]['pk'],
> sp=params,
> total=10000, // Fungible tokens have totalIssuance greater than 1
> default_frozen=False,
> unit_name="ALICE",
> asset_name="Alice's Coins@arc3",
> manager="",
> reserve="",
> freeze="",
> clawback="",
> url="https://path/to/my/fungible/asset/metadata.json",
> metadata_hash=json_metadata_hash,
> decimals=2) // Fungible tokens typically have decimals greater than

[https:\\ALICE NFT.io](https://path/to/my/fungible/asset/metadata.json",  )
